common/dwr.c

```diff
@@ -1667,6 +1667,36 @@ static void dwr_speed_hacks(dw_rom *rom)
         /* speed up the text */
         vpatch(rom, 0x7a33, 3, 0xea, 0xea, 0xea);
     }
+	
+	IF (MEDIUM_TEXT(rom)) {
+		printf("Enabling medium text...\n");
+		
+        /* speed up the text, 2 characters / frame */
+        vpatch(rom, 0x7a44, 2, 0x0a, 0xbf);
+        vpatch(rom, 0x7f1a, 2, 0xa5, 0xd2, 
+			0x29, 0x01, 
+			0xd0, 0x03, 
+			0x20, 0x74, 0xff, 
+			0x60);
+	}
+	
+	if (DOUBLE_WALKING(rom)) {
+		printf("Doubling walking speed...\n");
+		
+		/* double walking speed */
+		vpatch(rom, 0x32a5, 2, 0x80, 0xa5);
+		vpatch(rom, 0x339f, 2, 0x80, 0xa5);
+		vpatch(rom, 0x3423, 2, 0x90, 0xa5);
+		vpatch(rom, 0x3439, 2, 0x90, 0xa5);
+		vpatch(rom, 0x3466, 2, 0x90, 0xa5);
+		vpatch(rom, 0x3490, 2, 0x90, 0xa5);
+		vpatch(rom, 0x3578, 2, 0x90, 0xa5);
+		vpatch(rom, 0x35a5, 2, 0x90, 0xa5);
+		vpatch(rom, 0x35cf, 2, 0x90, 0xa5);
+		vpatch(rom, 0x36f4, 1, 0x07);
+		vpatch(rom, 0x2590, 16, 0xA5, 0x45, 0xC9, 0x01, 0xD0, 0x06, 0xA5, 0x05, 0x29, 0x01, 0xD0, 0x03, 0x20, 0x74, 0xFF, 0x60);
+		vpatch(rom, 0x25a0, 16, 0xA5, 0x45, 0xC9, 0x01, 0xD0, 0x06, 0xA5, 0x07, 0x29, 0x01, 0xD0, 0x03, 0x20, 0x74, 0xFF, 0x60);
```

Comment:
These should have comments with assembly code which denotes the changes for easier maintenance. Examples can be seen elsewhere in the code.

Reply:
My bad; I added comments to the vpatches.